### DISCUSSION TITLE
Can't install Atom, says "Update has stopped working"

### DISCUSSION THREAD

**Discussion by FrostShock** [atom/atom#22745] [General]
I tried to install Atom 1.57 and 1.55 and I'm getting the same error: "Update has stopped working" - "A problem caused the program to stop working correctly. Windows will close the program and notify you if a solution is available." And then Atom says "Installation has failed" - "There was an error while installing the application. Check the setup log for more information and contact the author"

Then I push the "Open Setup Log" button and nothing happens.

Where is the setup log?

I am using Windows 8.1, I installed it two days ago from the DVD image I downloaded from Microsoft two days ago.

**Comment by cicatepit** [comment]
https://github.com/atom/atom/issues/20169#issuecomment-557119314

portable works like a charm, indeed.
